Healthy/Classes/Modules/Dashboard/Views/FoodTagsView/TagCollectionViewCell/TagCollectionViewCell.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+class TagCollectionViewCell: UICollectionViewCell {
+
+    // MARK: Outlets
+
+    @IBOutlet weak var backView: UIView! {
+        didSet {
+            backView.layer.cornerRadius = 10
+        }
+    }
+    @IBOutlet weak var foodCategoryName: UILabel!
+
+    func setFoodCategoryName(_ title: String) {
```

Comment:
We need to create a view model for this cell to follow the convention. 
```suggestion
    func updateView(viewModel: ViewModel) {
```